src/App.tsx

```diff
@@ -15,48 +17,113 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
```

Comment:
```suggestion
enum SortType {
```

Reply:
<img width="748" alt="image" src="https://github.com/mate-academy/react_list-of-goods-hooks/assets/86789161/03682096-f63a-4fbd-a4cb-6daae14f308f">